tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -0,0 +1,40 @@
+""" Unit tests for Riemannian Mean Shift method """
+
+import numpy as np
```

Comment:
Remove this line and instead:
`import geomstats.backend as gs`

and on line 16, use:
`gs.random.seed(10)`